README.md

```diff
@@ -95,7 +97,7 @@ but with the default settings it can produce different tokens from MeCab.
 
 For example, MeCab ignores spaces (more precisely, `SPACE` defined in `char.def`) in tokenization.
 
-```
+```shell
```

Comment:
This block contains the result of the command, and that result is highlighted incorrectly.
Blocks should be left as is.

Reply:
I reverted it for code blocks containing output results. https://github.com/daac-tools/vibrato/pull/112/commits/8cdc1ec4d8d3720f3ade4b9ae5b2dc12b30eb41d